.circleci/config.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: hashicorop/terraform:0.8.2
```

Comment:
I've updated the version to 0.9.11 and fixed the name to properly point to hashicorp  🙇 .    This change is only to address validations on pull requests for this repo.